myapp/forms.py

```diff
@@ -0,0 +1,35 @@
+#__author__ = Mohammad Abdin
+from django import forms
+from .models import Profile
+from django.contrib.auth.models import User
+
+
+
+""" this is the form for updating the user's profile."""
```

Comment:
you can add to the docstring, called from http://127.0.0.1:8000/accounts/profile/edit page by clicking the update submit button